### PR TITLE
🎚Lint Alignment: Align the extension and `@thesis-co/eslint-config` styleguides

### PR DESCRIPTION
**Pull request by mhluongo**
Does what it says on the tin, fixing some inconsistencies around quote, arrow function, and semi-colon usage and removing a few dev dependencies to boot.

Leaving this in draft until the relevant [`@thesis-co/eslint-config` PR](https://github.com/thesis/eslint-config/pull/7) lands and a new NPM package is published.

# To Test

- [ ] Review whether these changes leave any rules unaccounted for (eg, the linter was accidentally relaxed)

### CODE REVIEW

File: background/lib/token-lists.ts [hunk 1]
```diff
@@ -66,19 +66,17 @@ function tokenListToFungibleAssetsForNetwork(
...
-          tokenLists: [tokenListCitation],
-        },
-        name: tokenMetadata.name,
-        symbol: tokenMetadata.symbol,
-        decimals: tokenMetadata.decimals,
-        homeNetwork: network,
-        contractAddress: tokenMetadata.address,
-      }
-    })
+    .map((tokenMetadata) => ({
```
Comment (Shadowfiend) [inline background/lib/token-lists.ts:69]: This is a weird one. What rule did we change?

Comment (mhluongo) [reply background/lib/token-lists.ts:69]: [`arrow-body-style`](https://eslint.org/docs/latest/rules/arrow-body-style) I believe

Comment (Shadowfiend) [reply background/lib/token-lists.ts:69]: Did we change it explicitly or did a change get pulled in from somewhere? If the former, let's unchange.

**Comment by Shadowfiend** [comment]
Closing in favor of #3571.

**Pull request closed without merging**